app/src/main/java/com/android/voyageur/ui/trip/activities/Activities.kt

```diff
@@ -94,23 +135,36 @@ fun ActivitiesScreen(
             selectedItem = navigationActions.currentRoute(),
             userViewModel = userViewModel)
       },
-      topBar = { // TODO: include the Search in TopAppBar
+      topBar = {
         TopAppBar(
             title = {
-              Text("Activities bar")
-            }, // This is a hardcoded string, as it should be replaced by the search bar
+              Box(
```

Comment:
I agree with Andreea, the search field is a bit too big, it leaves little room for scrolling for activities. Try reducing the padding for the start? This might be enough, since now there's quite some unused space because of the padding

Reply:
Thanks for the suggestion! I agree with you both. As for the previous suggestions, I'll convert it to an issue and solve it in the UI/UX PR I'm working on.

Reply:
Thanks for your input! Since this fix is directly related to the current PR, I think it would be better to address it here than in the UI/UX one. That PR should be for fixes for PRs that have already been merged. 
As for my other comment, I think that it should be done either here or in a new PR, not in the UI/UX one since it's a feature, and not UI related. What do you think?

Reply:
As requested, I changed the padding and size of the search bar in the Activities screen, closing the issue #272. 
For the search bar in the ByDay screen, I think it could be added in the UI/UX PR since it would not be a new feature but an improvement on the UX for the ByDay screen. Let me know what you think :)

Reply:
Thanks for working on fixing the padding, however for me it still looks like not much changed. Is this the intended behavior?
![search bar](https://github.com/user-attachments/assets/a2d4ac1b-ba73-45b8-b2b0-9f3ed4c49f6a)


 I still think that the search bar on the ByDay screen belongs better in this PR than the UI/UX one (since I was thinking that that one should be more about little improvements, not new working components). But, if you strongly believe that should be the case, I leave it up to you.